convert2rhel/actions/pre_ponr_changes/backup_system.py

```diff
@@ -39,15 +39,18 @@ def run(self):
             # https://github.com/oamg/convert2rhel/blob/v1.2/convert2rhel/subscription.py#L189-L200
             system_release_file.backup()
             os_release_file.backup()
-        except SystemExit as e:
-            # TODO(pr-watson): Places where we raise SystemExit and need to be
-            # changed to something more specific.
-            # Raised in module redhatrelease on lines 49 and 60
-            #   - If unable to find the /etc/system-release file,
-            #     SystemExit is raised
+
+        except exceptions.CriticalError as e:
             self.set_result(
-                level="ERROR", id="UNKNOWN_ERROR", title="An unknown error has occurred", description=str(e)
+                level="ERROR",
+                id=e.id,
+                title=e.title,
+                description=e.description,
+                diagnosis=e.diagnosis,
+                remediation=e.remediation,
+                variables=e.variables,
             )
+        return
```

Comment:
Do we need this empty return here?

Reply:
We don't.  I was contemplating whether it made sense to have a return there somewhere since we would need to explicitly return when we encounter the error if we add additional code after the exception block at a later time and it didn't feel right to have a return inside the block, after `set_result()` and not also at the end.

I could remove this `return` and trust that the person who does that will remember to put it in or I could add a second `return` inside of the exception block to reinforce what's happening.

What do you think?

Reply:
I think we can remove it for now and leave that for future-someone who will work on that action. 

I don't think we have to take care of that right away.

Reply:
Now removed!